tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -454,8 +454,6 @@ def _execute_test_suite(self, suite: TestSuiteInfo) -> bool:
         suite_full_name = f"{suite_name}-{self.context.environment_name}"
         suite_start_time: datetime.datetime = datetime.datetime.now()
 
-        success: bool = True  # True if all the tests succeed
```

Comment:
this variable was a leftover of a previous iteration of the code and _execute_test_suite was always returning True (i.e. Success, so VM logs were not being collected on test failures)